parts/chapters/subsections/4.3/SKIP300.fodt

```diff
@@ -4205,12 +4204,12 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
-    <text:p text:style-name="P59">All keywords between the SKIP<text:span text:style-name="T20">3</text:span>00 and ENDSKIP keywords are currently also ignored by OPM Flow. </text:p>
-    <text:p text:style-name="P59">The nesting of pairs of either the SKIP, SKIP100 or SKIP300 keyword and the ENDSKIP keyword is not supported by OPM Flow.</text:p>
+    <text:p text:style-name="P59">All keywords between the <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP_11_3">SKIP</text:a><text:span text:style-name="T20">3</text:span>00 and <text:a xlink:href="#__RefHeading___Toc605472_3199477706">ENDSKIP</text:a> keywords are currently also ignored by OPM Flow. </text:p>
+    <text:p text:style-name="P59">The nesting of pairs of either the <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP_11_3">SKIP</text:a>, <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP100_11_3">SKIP100</text:a> or <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP300_11_3">SKIP300</text:a> keyword and the <text:a xlink:href="#__RefHeading___Toc605472_3199477706">ENDSKIP</text:a> keyword is not supported by OPM Flow.</text:p>
     <text:h text:style-name="Heading_20_4" text:outline-level="4" text:is-list-header="true"><text:bookmark-start text:name="__RefHeading___Toc14501_37011683883711111"/>Example<text:bookmark-end text:name="__RefHeading___Toc14501_37011683883711111"/></text:h>
-    <text:p text:style-name="P60">If the following example is read using the commercial <text:span text:style-name="T20">compositional</text:span> simulator then the SKIP<text:span text:style-name="T20">3</text:span>00 keyword will cause the black-oil PVT data to be ignored, <text:span text:style-name="T20">the input will continue to be read after the next ENDSKIP keyword is encountered</text:span>, <text:span text:style-name="T20">and</text:span> the SKIP<text:span text:style-name="T20">1</text:span>00 keyword will be ignored and the compositional PVT data will be read <text:span text:style-name="T20">along with</text:span> the remainder of the <text:span text:style-name="T20">input</text:span>.</text:p>
+    <text:p text:style-name="P60">If the following example is read using the commercial <text:span text:style-name="T20">compositional</text:span> simulator then the <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP_11_3">SKIP</text:a><text:span text:style-name="T20">3</text:span>00 keyword will cause the black-oil PVT data to be ignored, <text:span text:style-name="T20">the input will continue to be read after the next <text:a xlink:href="#__RefHeading___Toc605472_3199477706">ENDSKIP</text:a> keyword is encountered</text:span>, <text:span text:style-name="T20">and</text:span> the <text:a xlink:href="#REF_HEADING_KEYWORD_SKIP_11_3">SKIP</text:a><text:span text:style-name="T20">1</text:span>00 keyword will be ignored and the compositional PVT data will be read <text:span text:style-name="T20">along with</text:span> the remainder of the <text:span text:style-name="T20">input</text:span>.</text:p>
```

Comment:
I think here the initial formatting `SKIP<text:span ..>3</text:span>00` broke the link. The some links  should go to SKIP300 and not SKIP